platforms/ios/lib/WysiwygComposer/Sources/WysiwygComposer/Components/WysiwygComposerView/WysiwygComposerViewModel.swift

```diff
@@ -211,8 +211,9 @@ private extension WysiwygComposerViewModel {
     ///   - end: End location for the selection.
     func applyReplaceAll(codeUnits: [UInt16], start: UInt32, end: UInt32) {
         do {
-            let html = String(utf16CodeUnits: codeUnits,
-                              count: codeUnits.count)
+            let htmlFragment = String(utf16CodeUnits: codeUnits,
+                                      count: codeUnits.count)
+            let html = generateHtmlBodyWithStyle(htmlFragment: htmlFragment)
```

Comment:
I think this should only be applied at the UI level, as I expect this property to store the HTML we intend to send in the actual event formatted_body at some point.

Reply:
😂 Yea that wasn't particularly thought through. Thanks